hathor/cli/side_dag.py

```diff
@@ -136,14 +135,14 @@ def main(capture_stdout: bool) -> None:
 
 def _process_logging_output(argv: list[str]) -> tuple[LoggingOutput, LoggingOutput]:
     """Extract logging output before argv parsing."""
-    from hathor.cli.util import LoggingOutput
+    from hathor.cli.util import LoggingOutput, create_parser
 
     class LogOutputConfig(str, Enum):
         HATHOR = 'hathor'
         SIDE_DAG = 'side-dag'
         BOTH = 'both'
 
-    parser = argparse.ArgumentParser(add_help=False)
+    parser = create_parser(add_help=False)
```

Comment:
The main difference is that `create_parser` will also consider environment variables, this is intentional, right? I think it should be added to the acceptance criteria.